BLSQ/iaso
# IA-2853: public registry front part 2, the registry

Target here is to make the public registry page work using public registry url.
It should also use own hook in order to tweak calls later with new api.

Related JIRA tickets : IA-2853
## Self proofreading checklist


## Doc
--

## Changes

- Copy registry components using hooks or `braseUrls` into public registry folder
- copy hooks
- remove all optional components using permission to be displayed, ad we don't have user
- own `breadcrumb` and `linkToRegistry` component
- fixed home page background issue 
- useStaticUrl hook 

## How to test

Make sure you are connected have registry plugins activated
adapt plugins/registry/js/src/domains/registry/index.tsx config with version and datasource ids you want to display
Visit http://localhost:8081/registry/data
Select a country and play with the registry.

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/228f8a6c-986d-4f68-a7da-5741ea9b82ec



## Notes

Next step will be the migration to the new api  from @mathvdh 
You have to run `npm ci` to upgrade `bluesquare-components`
